Source/Fuse.Reactive.Bindings/Instance.API.uno

```diff
@@ -0,0 +1,356 @@
+using Uno;
+using Uno.Collections;
+using Uno.UX;
+
+using Fuse.Internal;
+
+namespace Fuse.Reactive
+{
+	public enum InstanceDefer
+	{
+		/** Items will be added immediately to the tree. */
+		Immediate,
+		/** Item changes will be buffered and added once per frame. */
+		Frame,
+		/** Items will be added as though they are wrapped in a @Deferred node. */
+		Deferred,
+	}
+	
+	public enum InstanceReuse
+	{
+		/** Instances are not reused */
+		None,
+		/** Instances can be reused in the same frame */
+		Frame,
+	}
+	
+	public enum InstanceObjectMatch
+	{
+		/** New objects are always new, never matched to an existing one */
+		None,
+		/** The `ObjectId` is used to compare objects. This is set implicitly if `ObjectId` is set. */
+		FieldId,
+		/** Use the object itself as the matching key. Suitable for when the object is a plain string or number. */
+		Object,
+	}
+	
+	/* The protected and public API of the instantiator. */
+	public partial class Instantiator
+	{
+		IList<Template> _templates;
+		RootableList<Template> _rootTemplates;
+		
+		/** Specifies a list of templates that will be used to reflect the data in `Items`.
+
+			Typically, this collection is not referred to directly. Rather, it will contain all of the children of the `Each` tag in UX.
+		*/
+		[UXPrimary]
+		public IList<Template> Templates
+		{
+			get
+			{
+				if (_templates != null)
+					return _templates;
+					
+				_rootTemplates = new RootableList<Template>();
+				if (IsRootingCompleted)
+					_rootTemplates.Subscribe(OnTemplatesChanged, OnTemplatesChanged);
+				_templates = _rootTemplates;
+				return _templates;
+			}
+		}
+
+		InstanceDefer _defer = InstanceDefer.Frame;
+		/** Defers the creation items to avoid processing bottlenecks.
+		
+			The default is `Frame`.
+		*/
+		public InstanceDefer Defer
+		{
+			get { return _defer; }
+			set { _defer = value; }
+		}
+		
+		InstanceReuse _reuse = InstanceReuse.None;
+		/** Attempts to reuse template instances when items are being removed and created.
+		
+			The default is `None`
+			
+			Be aware that when using this feature several other features may no longer work as expected, such as:
+				- RemovingAnimation: the reused items are not actually removed
+				- AddingAnimation: the resused items are not actually added, just moved
+				- Completed: As a reused item is not added/removed it will not trigger a second time
+				
+			This feature will remain experimental until we can figure out which of these issues can be solved, avoided, or just need to be accepted.
+				
+			@experimental
+		*/
+		public InstanceReuse Reuse
+		{
+			get { return _reuse; }
+			set { _reuse = value; }
+		}
+		
+		InstanceObjectMatch _objectMatch = InstanceObjectMatch.None;
+		/**
+			Reuses existing nodes if the new objects match the old ones.
+			
+			This field is typically set implicity. It defaults to `None`. Use `ObjectId` instead if you want to match based on a id field. 
```

Comment:
The default is `None` and using an `ObjectId` will set this to `Field`. Only the case where you want the object to match itself is this property specifies explicitly.